assets/rbac/node_role.yaml

```diff
@@ -3,28 +3,13 @@ kind: ClusterRole
 metadata:
   name: shared-resource-secret-configmap-share-watch-sar-create
 rules:
-  - apiGroups:
-      - ""
-    resources:
-      - secrets
-      - configmaps
-      - pods
-    verbs:
-      - get
-      - list
-      - watch
-  - apiGroups:
-      - sharedresource.openshift.io
-    resources:
-      - sharedconfigmaps
-      - sharedsecrets
-    verbs:
-      - get
-      - list
-      - watch
-  - apiGroups:
-      - authorization.k8s.io
-    resources:
-      - subjectaccessreviews
-    verbs:
-      - create
\ No newline at end of file
+  - apiGroups: [""]
+    resources: ["secrets"]
+    resourceNames: ["example-secret1", "example-secret2", "example-secret3"]
+    verbs: ["get", "list", "watch"]
+  - apiGroups: ["sharedresource.openshift.io"]
+    resources: ["sharedconfigmaps", "sharedsecrets"]
+    verbs: ["get", "list", "watch"]
+  - apiGroups: ["authorization.k8s.io"]
+    resources: ["subjectaccessreviews"]
```

Comment:
And we should check whether anything is broken if SAR is removed from permission.